bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -91,7 +101,9 @@ public void onFinished() {
             }
         });
 
-        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
+        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
+        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
```

Comment:
This can be very misleading. The request id is that of a binding, not of a discovery service. So to me returning a DiscoveryServiceDTO is simply wrong.